Classes/Settings/SettingsViewController.swift

```diff
@@ -84,15 +84,18 @@ GitHubSessionListener {
             case .success(let response):
                 let text: String
                 let color: UIColor
-                switch response.data.status {
-                case .good:
-                    text = NSLocalizedString("Good", comment: "")
+                switch response.data.status.indicator {
+                case .none:
+                    text = response.data.status.description
```

Comment:
This should be able to be pulled out of the switch (DRY).

Reply:
Sorry for this mistake! Will fix it and thank you for the review!